hedera-node/hedera-smart-contract-service/src/main/java/com/hedera/node/app/service/contract/ContractService.java

```diff
@@ -29,6 +29,20 @@
 public interface ContractService extends Service {
     String NAME = "ContractService";
 
+    /**
+     * {@inheritDoc}
+     *
+     * Ensure the contract service schemas are migrated before the {@code TokenService} schemas, since
+     * the {@code TokenService} depends on the {@code ContractService} to know the updated first storage
+     * keys for contracts with broken storage links.
+     *
+     * @return {@code Integer.MIN_VALUE}
+     */
+    @Override
+    default int migrationOrder() {
+        return Integer.MIN_VALUE;
```

Comment:
Interesting ... 🤔 . Do you think this migrationOrder matters for any other services? Probably `FileService` should be done first before everything?

Reply:
Added comment to clarify.